File: TableViewRefresh/TableViewRefresh/ViewController.m
//
//  ViewController.m
//  TableViewRefresh
//
//  Created by 高李军 on 15/12/4.
//  Copyright © 2015年 高李军. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.

    self.refreshControl = [[UIRefreshControl alloc]init];
    
    [self.refreshControl beginRefreshing];
    
    
}

-(void)viewWillAppear:(BOOL)animated{
    [super viewWillAppear:animated];

}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: 自定义控制器视图的转场动画_Swift/自定义控制器视图的转场动画_Swift/ViewController.swift
//
//  ViewController.swift
//  自定义控制器视图的转场动画_Swift
//
//  Created by 高李军 on 15/11/25.
//  Copyright © 2015年 高李军. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet weak var lblMessage:UILabel!
    
    

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        let swipeGestureRecongnizer:UITapGestureRecognizer = UITapGestureRecognizer(target: self, action: "showSecondViewController");
        self.view.addGestureRecognizer(swipeGestureRecongnizer);
        
    }
    
    func showSecondViewController(){
        self.performSegueWithIdentifier("idFirstsegue", sender: self);
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

